pkgs/os-specific/linux/amdgpu-pro/default.nix

```diff
@@ -22,149 +22,123 @@ let
       "x86_64-linux-gnu"
     else throw "amdgpu-pro is Linux only. Sorry. The build was stopped.";
 
-  libReplaceDir = "/usr/lib/${libArch}";
-
   ncurses5 = ncurses.override { abiVersion = "5"; };
 
 in stdenv.mkDerivation rec {
 
-  version = "17.40";
+  version = "21.30";
   pname = "amdgpu-pro";
-  build = "${version}-492261";
+  build = "${version}-1290604";
 
-  libCompatDir = "/run/lib/${libArch}";
+  outputs = [ "out" ] ++ optionals (!libsOnly) [ "fw" "kmod" ];
```

Comment:
I'm not so sure this is the best way, but using $out for these seemed to pollute the system.